blocks/hooks/align.js

```diff
@@ -131,7 +131,7 @@ export function withAlign( BlockListBlock ) {
 export function addAssignedAlign( props, blockType, attributes ) {
 	const { align } = attributes;
 
-	if ( includes( getBlockValidAlignments( blockType ), align ) ) {
+	if ( align && includes( getBlockValidAlignments( blockType ), align ) ) {
```

Comment:
I think this is where `alignundefined` could be generated before this commit.

Reply:
Ooops, you are right. It is obsolete unless we want to add performance optimization to avoid array lookup :P I'm removing this change.